### PR TITLE
adds model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb

### PR DESCRIPTION
**Pull request by kirbyju**
### Description
I made the summary more focused on the specific prostate dataset and model being used and also consolidated the setup steps (installations/imports) at the top.

### Checks

- [x] Avoid including large-size files in the PR.
- [x] Clean up long text outputs from code cells in the notebook.
- [x] For security purposes, please check the contents and remove any sensitive info such as user names and private key.
- [x] Ensure (1) hyperlinks and markdown anchors are working (2) use relative paths for tutorial repo files (3) put figure and graphs in the `./figure` folder
- [x] Notebook runs automatically `./runner.sh -t <path to .ipynb file>`

### CODE REVIEW

**Comment by wyli** [comment]
thanks @kirbyju, now the tests work fine, I mainly
- added a license cell
- updated the import statements (removed most of the `--pre` flag)
- removed `display(df)` as it raises an error "stdin:154:1: F821 undefined name 'display'
display(df)"

please help confirm those changes.

**Comment by wyli** [comment]
thanks, let's wait for Mingxin's review, then I'll update and merge it.

File: model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb
Comment (mingxin-zheng) [inline model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb:212]: Thank you @kirbyju for preparing the PR! Below are my comments:
Typo:"prevoiusly"

Comment (mingxin-zheng) [reply model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb:212]: The link ("https://doi.org/10.7937/K9TCIA.2017.MURS5CL10.7937/TCIA.X0H0-1706") does not work for me. Can you please check? Thanks!

Comment (mingxin-zheng) [inline model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb:212]: I think the sentence need the article "the".
could be used with **the** Prostate MRI

File: model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb
Comment (mingxin-zheng) [inline model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb:113]: `pydicom` is included in dev env since monai 0.9.1. Maybe we can move the installation to
```
pip install monai[..., pydicom]
```

File: model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb
Comment (mingxin-zheng) [reply model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb:212]: The words "our APIs" are a bit confusing to me. Who are "we" here?

File: model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb
Comment (mingxin-zheng) [inline model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb:329]: `zip_dir` seems unreferenced. Maybe `zoo_dir`?

File: model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb
Comment (mingxin-zheng) [inline model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb:188]: I think the reader may want to read this markdown section after they install the imjoy extension and before they run the imports. How about putting this with the "## Setup environment " markdown block, for example:
```markdown
## Setup environment
NOTE: On many systems you must manually install the imjoy-jupyter-extension for itkWidgets!
If you do not see a blue 'ImJoy' icon on the menu bar in this notebook:

Google CoLab: The following does not apply to Google CoLab - it will not show an ImJoy and all should work without modification.
Enable Extensions: Many Jupyter Lab systems disable jupyter extensions by default, and they must be enabled for this notebook to work. Typically this is accomplished using the Jupyter interface to select the extension manager (left-hand side, icon that looks like a piece of a puzzle) and select the Enable button if it appears.
Install imjoy extension: In the extension manager, search for 'imjoy' and install the 'imjoy-jupyter-extension'. The installation can take several minutes. It may also prompt you to rebuild, save, and reload your jupyter environment as part of this process. In the end, you should see a blue 'ImJoy' icon on the left side of the menu bar in this notebook.
```

File: model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb
Comment (kirbyju) [reply model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb:212]: Good catches, thanks.

Comment (kirbyju) [reply model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb:212]: I'll clarify that it's the TCIA APIs.

File: model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb
Comment (kirbyju) [reply model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb:113]: I defer to you all here.  Does monai also include pydicom-seg or just pydicom?  Could both be moved into the monai install command?

File: model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb
Comment (kirbyju) [reply model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb:329]: I've asked Stephen Aylward to take a look at this discussion.  I'm not sure what he intended here.

File: model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb
Comment (kirbyju) [reply model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb:188]: I think it'd be good for Stephen to chime in here too.

**Comment by kirbyju** [comment]
On Google Colab I get the following warnings/errors when I run the Setup step, but it still seems to let me run all of the subsequent steps in the notebook successfully so maybe it's not a problem?

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'monai'
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
pydicom-seg 0.4.0 requires jsonschema<4.0.0,>=3.2.0, but you have jsonschema 4.17.3 which is incompatible.
google-colab 1.0.0 requires notebook~=5.7.16, but you have notebook 6.5.2 which is incompatible.
google-colab 1.0.0 requires tornado~=6.0.4, but you have tornado 6.2 which is incompatible.
flask 1.1.4 requires Jinja2<3.0,>=2.10.1, but you have jinja2 3.1.2 which is incompatible.
```

File: model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb
Comment (wyli) [reply model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb]: I think it's ok to least it as it is for now as monai doesn't currently have a specific requirement for pydicom versions...

File: model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb
Comment (wyli) [reply model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb]: thanks I moved this cell up to the #Setup section.

File: model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb
Comment (wyli) [reply model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb]: I've changed that to zoo_dir for now, please feel free to revise.

**Review by wyli — APPROVED** [review]
thanks, I'm merging this for now, please feel free to raise new PRs to revise it.

**Pull request merged**
